agents/lakeshore372/LS372_agent.py

```diff
@@ -977,6 +985,79 @@ def get_still_output(self, session, params=None):
 
         return True, "Current still output is {}".format(still_output)
 
+    def input_configfile(self, configfile, filepath='/ls372configs/'):
```

Comment:
Since this isn't a Agent Task or Process, and is only called internally within the Agent, we should mark it as "private" by naming it with a leading underscore, i.e. `_input_configfile()`. This'll keep it off the automatically generated list of tasks/processes in the Agent docs.

Reply:
An update to this after the call (and to summarize @mhasself's point) -- this could be made into a Task so that we can run it separately to re-establish the device state. In which case, we don't want to mark this as private.